tests/test_data_schema.py

```diff
@@ -23,10 +23,13 @@
 import time
 import unittest
 import requests
+import rustworkx as rx
```

Comment:
no need to import networkx and rustowrkx again in test file

Reply:
ended up using these to assert that the schema graph is of the correct type when use_networkx flag is True/False